components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -276,4 +325,44 @@ class GeckoEngine(
             this.fontSizeFactor = it.fontSizeFactor
         }
     }
+
+    private fun ContentBlockingController.LogEntry.BlockingData.hasBlockedCookies(): Boolean {
+        return category == COOKIES_BLOCKED_BY_PERMISSION ||
```

Comment:
nit: Should we just use Event.COOKIES_BLOCKED_BY_PERMISSION etc so we don't have to import all of them?

Reply:
I think we are importing them individually. Or are you referring to another thing ? :)

![image](https://user-images.githubusercontent.com/773158/65454972-3582fd80-de14-11e9-906b-0c71336b34b4.png)

Reply:
Fixed!